src/cdk/v2/destinations/eloqua/procWorkflow.yaml

```diff
@@ -0,0 +1,58 @@
+bindings:
+  - name: EventType
+    path: ../../../../constants
+  - path: ../../bindings/jsontemplate
+    exportAll: true
+  - path: ./config
+  - name: removeUndefinedAndNullValues
+    path: ../../../../v0/util
+  - name: defaultRequestConfig
+    path: ../../../../v0/util
+  - path: ./utils
+
+steps:
+  - name: messageType
+    template: |
+      .message.type.toLowerCase()
+  - name: validateInput
+    template: |
+      let messageType = .message.type;
```

Comment:
let messageType = .message.type.toLowerCase();
// this won't cause any error if message.type is not present

Reply:
it will always be present